.github/workflows/release.yml

```diff
@@ -34,22 +34,16 @@ jobs:
         env:
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: "1"
 
-      - name: build
+      - name: npm build
         run: |
           npm run build
 
       - name: publish to npm
         run: npm publish
         env:
-          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: release to github
```

Comment:
sorry i missed this one earlier, i plan on manually doing release+tag at the same time from the browser

mostly because it me write the change log manually, and can do it from my phone!

but we could also switch to semantic git commits and auto change log? then release would just be `git tag ... && git push --tags`

Reply:
> but we could also switch to semantic git commits and auto change log? then release would just be `git tag ... && git push --tags`

I'm inclined to it.

Reply:
We can merge this pr first and then try to automate the github release with the changelog.

Reply:
Sounds good